### PR TITLE
Replace wiki links with FG links

### PR DESCRIPTION
**Pull request by aaronyan**
Fixes https://github.com/Automattic/abacus/issues/717.

This replaces all the wiki links with FieldGuide links and is the final step in the wiki to FG migration (696-gh-Automattic/experimentation-platform). 

The pages with wiki links were found using this search: https://github.com/Automattic/abacus/search?q=wiki

### CODE REVIEW

**Review by jessie-ross — APPROVED** [review]
Thanks for updating these! 🚀

File: src/components/experiments/wizard/Beginning.tsx [hunk 1]
```diff
@@ -34,10 +34,23 @@ const Beginning = (): JSX.Element => {
...
+        </Link>{' '}
+        is a great place to start, it will instruct you on{' '}
+        <Link
+          underline='always'
+          href='https://fieldguide.automattic.com/the-experimentation-platform/document-experiment/'
+          target='_blank'
+        >
+          documenting your experiment
+        </Link>{' '}
+        and creating a P2 post.
```
Comment (razvanpapadopol) [inline src/components/experiments/wizard/Beginning.tsx:53]: Love the added extra bits of guidance! ❤️

File: src/lib/analyses.test.ts [hunk 2]
```diff
@@ -472,12 +471,9 @@ describe('getExperimentHealthIndicators', () => {
...
           },
-          "link": "https://github.com/Automattic/experimentation-platform/wiki/Experiment-Health#experiment-run-time",
+          "link": "https://fieldguide.automattic.com/the-experimentation-platform/experiment-health/#experiment-run-time",
           "name": "Experiment run time",
           "unit": "days",
-          "value": ${expectedExperimentRunTimeDays},
-        },
-      ]
-    `)
+          "value": `)
```
Comment (razvanpapadopol) [inline src/lib/analyses.test.ts]: This should fix the failing unit test:
```suggestion
          "value": ${experimentRunTimeDays},
        },
      ]
    `)
```

File: src/lib/analyses.test.ts [hunk 3]
```diff
@@ -450,7 +450,6 @@ describe('getExperimentHealthIndicators', () => {
   it('should work for an experiment that ran too long', () => {
     const experimentRunTimeDays = 50
     const startDatetime = subDays(new Date(), experimentRunTimeDays)
-    const expectedExperimentRunTimeDays = differenceInHours(new Date(), startDatetime) / 24
```
Comment (razvanpapadopol) [inline src/lib/analyses.test.ts:453]: This was indeed not needed since we're explicitly defining the `experimentRunTimeDays` in the test setup.

**Review by razvanpapadopol — APPROVED** [review]
🚀

File: src/lib/analyses.test.ts (hunk 2 above)
Comment (aaronyan) [reply src/lib/analyses.test.ts]: Thanks for the suggestion! I ended up adding this back in https://github.com/Automattic/abacus/pull/729/commits/bc3a8a44ddbc89e5f25f409ee55a5f00118631da.

**Pull request merged**